WP_Auth0.php

```diff
@@ -120,6 +120,9 @@ public function init() {
 
 		$this->check_signup_status();
 
+		// Store nonce value before headers are sent
+		new WP_Auth0_State_Handler();
```

Comment:
why isn't this being assigned anywhere? If this is some kind of "static helper" maybe you'd want to have a singleton to access it instead of creating new instances.

Reply:
🤔